doc/source/ray-air/examples/tensorflow_image_example.md

```diff
@@ -0,0 +1,265 @@
+---
+jupytext:
+    text_representation:
+        extension: .md
+        format_name: myst
+kernelspec:
+    display_name: Python 3
+    language: python
+    name: python3
+---
+
+
+# Training a TensorFlow Classifier
+
+This tutorial demonstrates how to train an image classifier using the [Ray AI Runtime](air) (AIR).
+
+You should be familiar with TensorFlow before starting this tutorial. If you need a refresher, read TensorFlow's [Convolutional Neural Network](https://www.tensorflow.org/tutorials/images/cnn) tutorial.
+
+## Before you begin
+
+* Install the [Ray AI Runtime](air). You'll need Ray 1.13 or later to run this example.
+
+```{code-cell} python3
+!pip install 'ray[air]'
+```
+
+* Install `tensorflow` and `tensorflow-datasets`
+
+```{code-cell} python3
+!pip install tensorflow tensorflow-datasets
+```
+
+## Load and normalize CIFAR-10
+
+We'll train our classifier on a popular image dataset called [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html).
+
+First, let's load CIFAR-10 into a Ray Dataset.
```

Comment:
Can you be more specific here? for example, 
1. Why do this?
2. What exactly are you doing below (next level of detail?)